README.md

```diff
@@ -381,31 +381,27 @@ $ aws configure
 
 ### Windows Azure CLI and credentials
 
-You first need to install node.js and NPM.  This version of Perfkit Benchmarker
-is known to be compatible with Azure CLI version 0.10.4, and will likely work
-with any version newer than that.
+This version of Perfkit Benchmarker is known to be compatible with Azure CLI 
+version 2.0.75, and will likely work with any version newer than that.
 
-Go [here](https://nodejs.org/download/), and follow the setup instructions.
-
-Next, run the following (omit the `sudo` on Windows):
+Follow the instructions at https://docs.microsoft.com/en-us/cli/azure/install-azure-cli or 
+on Linux, run the following commands:
 
 ```bash
-$ sudo npm install azure-cli -g
-$ azure login
+$ curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
```

Comment:
We should probably suggest the more generic install from https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-linux
`curl -L https://aka.ms/InstallAzureCli | bash` 
But the reference to the main docs is good.

Reply:
I've updated to use the more generic install instruction